### PR TITLE
Circle: rerun the configuration dialog for CPAN.pm

### PR DESCRIPTION
**Pull request by maxim-belkin**
If we run `cpan` after installing `XML::Parser` via
`sudo apt-get install ...` we get the following error message:

```
Sorry, we have to rerun the configuration dialog for CPAN.pm due to
some missing parameters.
...
```

This PR reruns this configuration in an automated way.
Code snippet by Clayton Dukes found [here](http://stackoverflow.com/questions/3462058/how-do-i-automate-cpan-configuration)

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ dependencies:
   override:
     - sudo apt-get update
     - sudo apt-get install curl g++ git libperl-dev libxml-parser-perl libxml-sax-perl make python-dev ruby2.0 ruby2.0-dev subversion wbritish
+    - (echo y;echo o conf prerequisites_policy follow;echo o conf commit)|cpan
```
Comment (sjackman) [inline circle.yml]: Use instead
```sh
printf "y\no conf prerequisites_policy follow\no conf commit\n" | cpan
```
Please document in this issue what the questions are asking, and include a link to this issue in a comment.

Comment (maxim-belkin) [inline circle.yml]:
```
1. Would you like to configure as much as possible automatically? [yes]
2. What approach do you want?  (Choose 'local::lib', 'sudo' or 'manual')
 [local::lib]
3. Would you like me to automatically choose some CPAN mirror
sites for you? (This means connecting to the Internet) [yes]
```
So, I guess we can use `printf "\n\n\no conf prerequisites_policy follow\no conf commit\n" | cpan`

Comment (maxim-belkin) [inline circle.yml]: I will use `sudo` for the second question: it proceeds much faster
```
printf "yes\nsudo\nyes\no conf prerequisites_policy follow\no conf commit\n" | cpan
```

**Comment by maxim-belkin** [comment]
Questions that `cpan` asks during reconfiguration:
```
1. Would you like to configure as much as possible automatically? [yes]
2. What approach do you want?  (Choose 'local::lib', 'sudo' or 'manual')
 [local::lib]
3. Would you like me to automatically choose some CPAN mirror
sites for you? (This means connecting to the Internet) [yes]
```
Our answers:
```
1. yes
2. sudo
3. yes
```

File: circle.yml (hunk 1 above)
Comment (maxim-belkin) [inline circle.yml]: added comment

File: circle.yml [hunk 2]
```diff
@@ -19,6 +19,7 @@ dependencies:
   override:
     - sudo apt-get update
     - sudo apt-get install curl g++ git libperl-dev libxml-parser-perl libxml-sax-perl make python-dev ruby2.0 ruby2.0-dev subversion wbritish
+    - printf "yes\nsudo\nyes\no conf prerequisites_policy follow\no conf commit\n" | cpan # https://github.com/Linuxbrew/homebrew-core/pull/1614
```
Comment (rwhogg) [inline circle.yml]: Maybe reference the specific comment? https://github.com/Linuxbrew/homebrew-core/pull/1614#issuecomment-274313223

Comment (maxim-belkin) [inline circle.yml]: sounds reasonable. will do

**Comment by maxim-belkin** [comment]
@rwhogg done! thanks!
@sjackman awaiting your approval

**Comment by sjackman** [comment]
Please squash and merge. Thanks, Maxim!

**Pull request merged**